recipes/idna/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, copy
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+import os
+
+required_conan_version = ">=1.52.0"
+
+class IdnaConan(ConanFile):
+    name = "idna"
+    description = "C++ library implementing the to_ascii and to_unicode functions from the Unicode Technical Standard."
+    license = "Apache-2.0", "MIT"
```

Comment:
```suggestion
    license = "Apache-2.0 OR MIT"
```
Can be made a bit clearer as a SPDX license expression.